source/layouts/layout.erb

```diff
@@ -37,102 +37,436 @@
     <meta property="og:image" content="/images/datacite.png" />
     <meta property="og:type" content="blog" />
 
-    <link href='//fonts.googleapis.com/css?family=Raleway:400,600,400italic,600italic' rel='stylesheet' type='text/css'>
-        <link
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href='//fonts.googleapis.com/css2?family=Barlow%3Aital%2Cwght%400%2C100%3B0%2C200%3B0%2C300%3B0%2C400%3B0%2C500%3B0%2C600%3B0%2C700%3B0%2C800%3B0%2C900%3B1%2C100%3B1%2C200%3B1%2C300%3B1%2C400%3B1%2C500%3B1%2C600%3B1%2C700%3B1%2C800%3B1%2C900&#038;display=swap&#038;ver=6.6.2' rel='stylesheet' type='text/css'>
+    <link href='//fonts.googleapis.com/css2?family=DM+Sans%3Aital%2Cwght%400%2C400%3B0%2C500%3B0%2C700%3B1%2C400%3B1%2C500%3B1%2C700&#038;display=swap&#038;ver=6.6.2' rel='stylesheet' type='text/css'>
+    <link
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
       integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog=="
       rel="stylesheet"
       crossorigin="anonymous" />
-
       
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+3:ital,wght@0,200..900;1,200..900&display=swap" rel="stylesheet">
 
     <link href="<%= ENV['CDN_URL'] %>/stylesheets/datacite.css" rel='stylesheet' type='text/css'>
```

Comment:
The overrides you've provided work well for the elements currently used in the site. However, if we want to use other elements, we'll potentially have to add more CSS to maintain the same style. 

For example, I just adjusted the "Older Versions" page to use h2 and h3 instead of h4, and added links to the h3 headers. This meant I had to fix the a:hover color for h3 > a to override what was inherited from datacite.css.